terra_layer/urls.py

```diff
@@ -1,17 +1,31 @@
 from django.urls import path
 from rest_framework import routers
 
-from .views import LayerViewset, LayerViews
+from .views import LayerViewset, LayerView, SceneViewset
 
 app_name = 'terralayer'
 
 router = routers.SimpleRouter()
 
-router.register(r'', LayerViewset, base_name='layer')
+router.register(r'layer', LayerViewset, base_name='layer')
+router.register(r'scene', SceneViewset, base_name='scene')
+
+scene_list = SceneViewset.as_view({
+    'get': 'list',
+    'post': 'create'
+})
+
+scene_detail = SceneViewset.as_view({
+    'get': 'retrieve',
+    'put': 'update',
+    'patch': 'partial_update',
+    'delete': 'destroy'
+})
```

Comment:
This is not needed as you can use the router for that

Reply:
Done